src/DND.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
+
+class DND {
+  constructor() {
+    this.prevRow;
+  }
+
+  setEventListeners() {
+    let listItems = document.querySelectorAll('li');
+  
+    listItems.forEach((listItem) => {
+  
+      listItem.addEventListener('dragstart', (e) => this.start(e));
+      listItem.addEventListener('dragover', (e) => this.over(e));
+      listItem.addEventListener('drop', (e) => this.drop(e));
+  
+    });
+  }
+
+  start(e) {
+    this.prevRow = e.target;
+
+    let HTMLContent = e.target.innerHTML;
+    let checkboxStatus = e.target.querySelector('input').checked;
+
+    e.dataTransfer.setData('html-content', HTMLContent);
```

Comment:
* I suggest you export this file as it is supposed to be imported as a module [Check project description](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Add%202%20new,completed%3A%20true%20/%20false). as it is part of this project's [learning objectives ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Learning%20objectives,write%20modular%20JavaScript.)